File: C/factorial.c
#include<stdio.h>
int factorial(int);
void main()
{
 int num,fact;
 printf("ENTER A NUMBER TO FIND ITS FACTORIAL:");
 scanf("%d",&num);
 fact=factorial(num);//CALLING FACTORIAL FUNCTION
 printf("\n FACTORIAL OF %d: %u ",num,fact);
}
int factorial(int a)//TO FIND FACTORIAL
{
 int p;
 if(a==1)
  return(1);
 else
 {
   p=a*factorial(a-1);//USE OF RECURSION
   return(p);
 }
}

File: Python/factorial.py
def main():
	print('Enter a number to calculate its factorial:')
	num = int(input())

	fact = factorial(num)
	print(f'\nThe factorial of {num} is: {fact}')

def factorial(a):
	if (a==1):
		return 1
	else:
		num = a * factorial(a-1)
		return num

main()
